packages/trpc/src/pools.ts

```diff
@@ -146,14 +146,11 @@ export const poolsRouter = createTRPCRouter({
               })
               .filter((pool): pool is NonNullable<typeof pool> => !!pool);
 
-            // won't sort if searching
-            if (search) return marketIncentivePools;
```

Comment:
This is now enforced on the client? I think it would be good for sorting to be possible while searching, but at least for the pools table in the pools page, having a rank-order fuzzy search result + sorting can yield what appear to be inaccurate results, especially for lower cap tokens (that have a strong search match but get sorted to the bottom)

Reply:
YE, I'll roll it back and implement it in a better way so we don't touch the current behaviour